citrination_client/client.py

```diff
@@ -271,19 +271,78 @@ def upload_file(self, file_path, data_set_id, root_path=None):
                 message = {"message": "Upload failed.",
                                    "status": r.status_code}
                 return json.dumps(message)
+        else:
+            return json.dumps({
```

Comment:
The return objects are a little hard to parse. Can you add another field, something like `"success": boolean`, that can be easily parsed by someone to know whether the upload was successful.